boards/arduino-uno/Makefile.include

```diff
@@ -0,0 +1,46 @@
+# define the cpu used by the arduino uno board
+export CPU = atmega328p
+
+export TERMPROG = $(RIOTBASE)/dist/tools/pyterm/pyterm
+export TERMFLAGS = -b 9600 -p $(PORT)
+
+#define the flash-tool and default port depending on the host operating system
+OS = $(shell uname)
+ifeq ($(OS),Linux)
+  PORT ?= /dev/ttyUSB0
```

Comment:
debian 8.4 here is the output of list-ttys.sh 
/sys/bus/usb/devices/1-1.2: FTDI FT232R USB UART serial: 'A6008bUi', tty(s): ttyUSB0
Not that I have an arduino duemilanove not an uno, maybe this is related....?